src/ImageSequencer.js

```diff
@@ -382,19 +382,6 @@ ImageSequencer = function ImageSequencer(options) {
     fs.writeFileSync('./src/Modules.js', mods);
   }
 
-  function createMetaModule(stepsCollection, info) {
```

Comment:
This will break the NDVI-Colormap module, which we should refactor to the new format.

Reply:
Or should we leave it like that as an example of expanding meta module?